DOCUMENTATION.md

```diff
@@ -9,5 +9,6 @@ Creates a new instance of `MeMarkdown`.
 - **Object** `options`: An object containing the following fields:
  - `events` (Array): An array with the events when the markdown code will be generated (default: `["input", "change"]`).
  - `callback` (Function): The callback function. If the second argument is a function, then it has greater priority.
+ - `toMarkdownOptions` (Object): Options to pass to the markdown converter code.
```

Comment:
Make sure to add this in the code as well. The `DOCUMENTATION.md` file is generated. :construction_worker: